### ISSUE TITLE
Test failure (with Graph 0.9721?)

### ISSUE THREAD

**Issue by eserte** [merkys/Chemistry-OpenSMILES#5]
The test suite fails on some of my smokers like this:
```
#   Failed test at t/06_write.t line 38.
#          got: 'N(C(Br)(O)(C))'
#     expected: 'N([C@](Br)(O)(C))'

#   Failed test at t/06_write.t line 43.
#          got: 'N(C(Br)(O)(C))'
#     expected: 'N([C@](Br)(O)(C))'

#   Failed test at t/06_write.t line 38.
#          got: 'N(C(Br)(O)(C))'
#     expected: 'N([C@@](Br)(O)(C))'

#   Failed test at t/06_write.t line 43.
#          got: 'N(C(Br)(O)(C))'
#     expected: 'N([C@@](Br)(O)(C))'
# Looks like you failed 4 tests of 22.
t/06_write.t .................. 
Dubious, test returned 4 (wstat 1024, 0x400)
Failed 4/22 subtests 
(etc)
```

Statistical analysis suggests that this happens with Graph 0.9721. Older and newer versions are OK:
```
****************************************************************
Regression 'mod:Graph'
****************************************************************
Name           	       Theta	      StdErr	 T-stat
[0='const']    	      1.0000	      0.0000	32648758624653052.00
[1='eq_0.9716']	      0.0000	      0.0000	   2.35
[2='eq_0.9721']	     -1.0000	      0.0000	-27315911257206832.00
[3='eq_0.9722']	     -0.0000	      0.0000	  -0.99

R^2= 1.000, N= 41, K= 4
****************************************************************
```

So maybe increase the prereq version for Graph?

**Comment by merkys** [comment]
Thanks for identifying the cause of the problem. Increasing prereq version for Graph is indeed the right solution here.

**Comment by merkys** [comment]
This issue is due to https://github.com/graphviz-perl/Graph/issues/22. Fixed by applying a workaround in 5a1943955c969092e22540d0b2c42ae4e0912226, thus now both earlier and later versions of Graph should work.
